src/ray/stats/metric_defs.cc

```diff
@@ -50,6 +50,66 @@ DEFINE_stats(grpc_server_req_handling, "Request number are handling in grpc serv
 DEFINE_stats(grpc_server_req_finished, "Finished request number in grpc server",
              ("Method"), (), ray::stats::COUNT);
 
+/// Object Manager.
+DEFINE_stats(object_manager_chunks_received_count,
+             "Number object chunks received broken per type {Total, FailedTotal, "
+             "FailedCancelled, FailedPlasmaFull}.",
+             ("Type"), (), ray::stats::GAUGE);
+
+/// Pull Manager
+DEFINE_stats(
+    pull_manager_bytes_usage,
+    "The total number of bytes usage broken per type {Available, BeingPulled, Pinned}",
+    ("Type"), (), ray::stats::GAUGE);
+DEFINE_stats(pull_manager_requested_bundles,
+             "Number of requested bundles broken per type {Get, Wait, TaskArgs}.",
+             ("Type"), (), ray::stats::GAUGE);
+DEFINE_stats(
```

Comment:
What's active bundle type?

Reply:
yeah it doesn't make much sense. Maybe we can separate out pull_manager_bundle_requests?

Reply:
Or we can make active get / wait / task args bundles instead (and merge this into pull_manager_requested_bundles)